modules/knock_down/action.sqf

```diff
@@ -0,0 +1,52 @@
+// AUTHOR: StatusRed (EM-Creations.co.uk)
+
+private _validWeapons = (_this select 0);
+private _reviveAction = (_this select 1);
+
+private ["_nearestunits", "_nearestunitofside", "_prevAnim"];
+
+_prevAnim = animationState player; // Get the player's current animation
+
+_nearestUnits = nearestObjects [player, ["Man"], 3]; // Get all units within range
+
+_useUnits = []; // Instanatiate an array for units which can be targetted
+
+// Check that they have a suitable weapon to knock down with
+if ((primaryWeapon player) in _validWeapons) then { // If the player has a valid weapon
+	if(count _nearestUnits > 0) then { // If there's at least one man nearby
+		{
+			_unit = _x;
+			//_isOut = _unit getVariable "knockedDown";
+			_isOut = false; // The above isn't working at the moment
+			if (((side player) != (side _unit)) && !_isOut) then { // If the unit is not the player, not on the same side and not already hit
+				_useUnits = _useUnits + [_unit]; // Add it to the targetable units array
+			};
+		} foreach _nearestUnits; // For each of the nearby men
+
+		if (count _useUnits > 0) then { // If there's at least one targetable unit
+
+			[-2, {
+				(_this select 0) switchMove "acts_miller_knockout";
+			}, [player]] call CBA_fnc_globalExecute; // Hitting animation
```

Comment:
we don't use `CBA_fnc_globalExecute` anymore we use the vanilla `remoteExec` or `cba_fnc_globalEvent`

Reply:
Done.